client/data/domains/forwarding/use-domain-forwarding-query.ts

```diff
@@ -1,42 +1,45 @@
-import { useQuery } from '@tanstack/react-query';
+import { UseQueryResult, useQuery } from '@tanstack/react-query';
 import wp from 'calypso/lib/wp';
 import { domainForwardingQueryKey } from './domain-forwarding-query-key';
 
-type DomainForwardingResponse = {
+export type DomainForwardingObject = {
 	domain_redirect_id: number;
 	domain: string;
 	subdomain: string;
 	target_host: string;
 	target_path: string;
-	forward_paths: '0' | '1';
-	is_secure: '0' | '1';
-	is_permanent: '0' | '1';
-	is_active: '0' | '1';
+	forward_paths: '0' | '1' | true | false;
+	is_secure: '0' | '1' | true | false;
+	is_permanent: '0' | '1' | true | false;
+	is_active: '0' | '1' | true | false;
```

Comment:
Could we coerce the api response into a boolean instead of mixing types like this? (I think you might be doing that already now?)

Reply:
That's a good idea, I'll fix the API.

Reply:
[Fixed it here](https://github.com/Automattic/wp-calypso/pull/81365/commits/8aea99b41c8471e7b9830f3f90483e07225c2530)